packages/stark-ui/src/modules/table/components/_table-theme.scss

```diff
@@ -0,0 +1,5 @@
+@mixin stark-table-theme($theme) {
+  .stark-table table {
+    border-radius: map-get($theme, border-radius);
+  }
+}
```

Comment:
I think those styles can be removed. As we discussed, we will only provided styles related to the color and font of the component in the theme SCSS.
For other styles, code should be placed in the _table.component.scss

In this specific case, we won't provide any border radius theme variable. If we want to change the angular material default radius, we should then put our style in the _table.component.scss. The end user can then either use it or not by importing this file.

Reply:
Ok, I'll remove them 😉